ai-ml/trainium-inferentia/variables.tf

```diff
@@ -1,6 +1,6 @@
 variable "name" {
   description = "Name of the VPC and EKS Cluster"
-  default     = "trainium-inferentia"
+  default     = "tr-inf"
```

Comment:
When making code alterations, particularly regarding renaming the `name`, it's crucial to verify that these changes do not disrupt existing docs or blogs. Please ensure you check the current blueprint document for any dependencies before proceeding. you can search the name to find the `trainium-inferentia` to find the occurances.

For example. you need to change this doc to reflect the new name https://github.com/awslabs/data-on-eks/blob/main/website/docs/gen-ai/inference/Llama2.md

and update this doc as well https://github.com/awslabs/data-on-eks/blob/main/website/docs/blueprints/ai-ml/trainium.md by replacing the `trainium` cluster name with new name

Should this be `trn1-inf2` instead?

Reply:
I will update the name to reflect trn1-inf2

Reply:
**NOTE:** Replace [cluster-name] with your actual EKS cluster name

Added this to the documentation because we cannot hardcode cluster name in the docs as we have added random string to the cluster name